src/BecklynSecurityBundle.php

```diff
@@ -16,10 +16,19 @@ public function build(ContainerBuilder $container)
     {
         parent::build($container);
 
-        $mappings = [
-            realpath(__DIR__ . '/../resources/config/doctrine-mapping') => 'Becklyn\Security\Infrastructure\Domain\Doctrine',
-        ];
-
-        $container->addCompilerPass(DoctrineOrmMappingsPass::createXmlMappingDriver($mappings));
+        $namespaces = ['Becklyn\Security\Infrastructure\Domain\Doctrine'];
+        $directories = [realpath(__DIR__ . '/Infrastructure/Store/Doctrine')];
+        $managerParameters = [];
+        $enabledParameter = false;
+        $aliasMap = [];
+        $container->addCompilerPass(
+            DoctrineOrmMappingsPass::createAnnotationMappingDriver(
+                $namespaces,
+                $directories,
+                $managerParameters,
+                $enabledParameter,
+                $aliasMap
+            )
+        );
```

Comment:
Huh, nice, I couldn't get the bundle to work with annotation mappings before so that's why I did it with xml mappings. If you switch to annotations successfully, you can also delete the resources/config/doctrine-mapping folder which contains the xml mappings which are no longer used.

As a matter of fact, can you look at if you could make this work with PHP8 attribute mappings instead of annotations?

Reply:
this should be working out of the box, also the directory is already deleted ;)

Reply:
Right, then make sure you test it on Genolotse :D